net/instaweb/rewriter/resource_fetch.cc

```diff
@@ -175,6 +175,7 @@ void ResourceFetch::HandleHeadersComplete() {
   for (int i = 0; i < driver_->options()->num_resource_headers(); ++i) {
     const RewriteOptions::NameValue* nv =
         driver_->options()->resource_header(i);
+    response_headers()->RemoveAll(nv->name);
     response_headers()->Add(nv->name, nv->value);
```

Comment:
Just using ->Replace might be a little nicer. Not that it's implemented any different